gravitee-apim-portal-webui-next/src/components/page/page.component.ts

```diff
@@ -15,14 +15,15 @@
  */
 import { Component, Input } from '@angular/core';
 
+import { PageAsciidocComponent } from './page-asciidoc/page-asciidoc.component';
 import { PageMarkdownComponent } from './page-markdown/page-markdown.component';
 import { PageSwaggerComponent } from './page-swagger/page-swagger.component';
 import { Page } from '../../entities/page/page';
```

Comment:
Idea: in the future we can consider creating aliases for main app directories, to keep it cleaner.